src/components/Products/ProductsList.vue

```diff
@@ -19,83 +19,35 @@
     >
     </products-list-item>
   </div>
+
+  <div v-else class="empty-product-list">
+    Product list is empty
```

Comment:
text should be inside inline html elements describing their content like span
`<strong>` would be a good choice here

Reply:
text should be translated